rpc/src/main/java/org/dst/rpc/core/exception/CodecException.java

```diff
@@ -0,0 +1,25 @@
+package org.dst.rpc.core.exception;
+
+
+public class CodecException extends RuntimeException {
```

Comment:
Use 2 exception instead?
`EncodingException ` and `DecodingException`. So that we can know what's wrong with it simply.

Reply:
I agree